Lib/test/test_importlib/namespace_pkgs/homonym/README.md

```diff
@@ -0,0 +1,2 @@
+This directory should not be a package, but should share a name with an
```

Comment:
Isn't there already a test directory for namespace packages? If so, we should just reuse it. And if there isn't, I _think_ the build infrastructure might need to be updated to know about the new directory (or at least it used to).

Reply:
Thanks for the review.

Happy to reuse any of the packages in this directory, however there are no name collisions between directories and submodules, so I don't see how I could reproduce the issue.

We do have several `foo.py` submodules here, so in the meantime I've changed this new directory to `foo` and removed the `homonym.py` file to remove 50% of the setup (although it's not as descriptive as it could be).

Let me know if I'm still missing your point!